src/components/views/LocalModList/SearchAndSort.vue

```diff
@@ -0,0 +1,132 @@
+<script lang="ts">
+import { Vue, Component } from 'vue-property-decorator';
+import { DeferredInput } from '../../all';
+import GameManager from '../../../model/game/GameManager';
+import { SortDirection } from '../../../model/real_enums/sort/SortDirection';
+import { SortLocalDisabledMods } from '../../../model/real_enums/sort/SortLocalDisabledMods';
+import { SortNaming } from '../../../model/real_enums/sort/SortNaming';
+import ManagerSettings from '../../../r2mm/manager/ManagerSettings';
+
+@Component({
+    components: {
+        DeferredInput,
+    }
+})
+export default class SearchAndSort extends Vue {
+    settings = new ManagerSettings();
+
+    get order() {
+        return this.$store.state.profile.order;
+    }
+
+    set order(value: SortNaming) {
+        this.$store.commit('profile/setOrder', value);
+        this.settings.setInstalledSortBy(value);
```

Comment:
I feel like the setting value change should be handled by the store rather than this component?